src/plugins/recordTypes/media/fields.js

```diff
@@ -0,0 +1,151 @@
+import { defineMessages } from 'react-intl';
+
+export default (configContext) => {
+  const {
+    AutocompleteInput,
+    CheckboxInput,
+    TextInput,
+    OptionPickerInput,
+  } = configContext.inputComponents;
+
+  const {
+    configKey: config,
+  } = configContext.configHelpers;
+
+  const {
+    DATA_TYPE_BOOL,
+  } = configContext.dataTypes;
+
+  return {
+    document: {
+      'ns2:media_common': {
+        identificationNumber: {
+          [config]: {
+            required: false,
+            view: {
+              type: TextInput,
+            },
+          },
+        },
+        title: {
+          [config]: {
+            defaultValue: 'media record',
```

Comment:
Make this 'Media record'. In the places this field is used, it makes more sense to be sentence case.